### DISCUSSION TITLE
The "path" argument must be of type string. Received null

### DISCUSSION THREAD

**Discussion by twincle** [pixijs/pixijs#8956] [Q&A]
### Current Behavior

<i> [webpack-dev-server] Project is running at:
<i> [webpack-dev-server] Loopback: http://localhost:3300/
<i> [webpack-dev-server] On Your Network (IPv4): http://192.168.7.106:3300/
<i> [webpack-dev-server] On Your Network (IPv6): http://[fe80::c972:a56c:6cf1:1d9e]:3300/
<i> [webpack-dev-server] Content not from webpack is served from 'D:\Code\BoomBeach\public' directory
assets by path *.js 1.33 MiB
  asset bundle.877531.js 1.28 MiB [emitted] [immutable] (id hint: vendors)
  asset bundle.443369.js 48.5 KiB [emitted] [immutable] (name: main)
asset 94fb23f66e50a3c9019a80708fde6953.png 750 bytes [emitted] [immutable] [from: src/bunny.png] (auxiliary name: main)
asset index.html 225 bytes [emitted]
Entrypoint main 1.33 MiB (750 bytes) = bundle.877531.js 1.28 MiB bundle.443369.js 48.5 KiB 1 auxiliary asset
runtime modules 29.4 KiB 14 modules
modules by path ./node_modules/@pixi/ 787 KiB 350 modules
modules by path ./node_modules/webpack-dev-server/client/ 53.5 KiB 12 modules
modules by path ./node_modules/webpack/hot/*.js 4.59 KiB 4 modules
modules by path ./node_modules/html-entities/lib/*.js 81.3 KiB 4 modules
modules by path ./src/ 1.51 KiB 3 modules
modules by path ./node_modules/url/ 37.4 KiB
  ./node_modules/url/url.js 22.8 KiB [built] [code generated]
  + 2 modules
modules by path ./node_modules/querystring/*.js 4.51 KiB
  ./node_modules/querystring/index.js 127 bytes [built] [code generated]
  + 2 modules
+ 5 modules

WARNING in ./node_modules/@pixi/settings/lib/external/ismobilejs/esm/index.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
The "path" argument must be of type string. Received null
TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received null
    at new NodeError (node:internal/errors:393:5)
    at validateString (node:internal/validators:163:11)
    at Object.isAbsolute (node:path:403:5)
    at fetchFromURL (D:\Code\BoomBeach\node_modules\source-map-loader\dist\utils.js:209:21)
    at D:\Code\BoomBeach\node_modules\source-map-loader\dist\index.js:105:41
    at Array.map (<anonymous>)
    at Object.loader (D:\Code\BoomBeach\node_modules\source-map-loader\dist\index.js:89:57)
 @ ./node_modules/@pixi/settings/lib/utils/isMobile.js 5:0-48
 @ ./node_modules/@pixi/settings/lib/index.js 6:15-45
 @ ./node_modules/@pixi/core/lib/index.js 10:15-40
 @ ./node_modules/pixi.js/lib/index.js 18:11-32
 @ ./src/index.ts 3:18-36

WARNING in ./node_modules/@pixi/settings/lib/utils/isMobile.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
The "path" argument must be of type string. Received null
TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received null
    at new NodeError (node:internal/errors:393:5)
    at validateString (node:internal/validators:163:11)
    at Object.isAbsolute (node:path:403:5)
    at fetchFromURL (D:\Code\BoomBeach\node_modules\source-map-loader\dist\utils.js:209:21)
    at D:\Code\BoomBeach\node_modules\source-map-loader\dist\index.js:105:41
    at Array.map (<anonymous>)
    at Object.loader (D:\Code\BoomBeach\node_modules\source-map-loader\dist\index.js:89:57)
 @ ./node_modules/@pixi/settings/lib/index.js 6:15-45
 @ ./node_modules/@pixi/core/lib/index.js 10:15-40
 @ ./node_modules/pixi.js/lib/index.js 18:11-32
 @ ./src/index.ts 3:18-36

WARNING in ./node_modules/@pixi/utils/lib/color/hex.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
The "path" argument must be of type string. Received null
TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received null
    at new NodeError (node:internal/errors:393:5)
    at validateString (node:internal/validators:163:11)
    at Object.isAbsolute (node:path:403:5)
    at fetchFromURL (D:\Code\BoomBeach\node_modules\source-map-loader\dist\utils.js:209:21)
    at D:\Code\BoomBeach\node_modules\source-map-loader\dist\index.js:105:41
    at Array.map (<anonymous>)
    at Object.loader (D:\Code\BoomBeach\node_modules\source-map-loader\dist\index.js:89:57)
 @ ./node_modules/@pixi/utils/lib/index.js 13:10-35
 @ ./node_modules/@pixi/core/lib/index.js 12:12-34
 @ ./node_modules/pixi.js/lib/index.js 18:11-32
 @ ./src/index.ts 3:18-36

webpack 5.75.0 compiled with 3 warnings in 3489 ms

### Environment

- **`pixi.js` version**: _7.0.4_
- **other**: _webpack 5.75.0   typescript 4.9.3_
- **Browser & Version**: _edge 108_
- **OS & Version**: _windows 11_

**Comment by bigtimebuddy** [comment]
These seems more like a Webpack issue. Moving to discussion. If you are able to isolate this as a PixiJS issue, please file a specific bug with a minimal reproduction. Thanks.

**Reply by twincle** [reply]
using the same webpack version, pixijs 6.x do not have these issues, but pixijs 7.x have ...
